.gitignore

```diff
@@ -5,5 +5,6 @@
 .env
 .envrc
 *.iml
+/iris-mpc-cpu/data
```

Comment:
Just a nit: can we move this into a new `.gitignore` file with entry `/data` in the `iris-mpc-cpu` subdirectory?  It doesn't seem like a specification that is relevant to the overall workspace.